target/linux/ramips/mt7620/profiles/zbt.mk

```diff
@@ -6,11 +6,11 @@
 #
 
 define Profile/ZBT-WE826
-	NAME:=ZBT-WE826 Device
+	NAME:=ZBT-WE826
 	PACKAGES:=\
-		kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
-		block-mount kmod-usb-core kmod-usb-dwc2 kmod-usb2 kmod-usb-ohci \
-		kmod-mt76
+		kmod-mt76 kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
+		block-mount kmod-usb-core kmod-usb2 kmod-usb-ohci kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-serial-wwan comgt wwan
```

Comment:
On 12/05/2016 10:04, Andrew Yong wrote:
> In target/linux/ramips/mt7620/profiles/zbt.mk
> <https://github.com/lede-project/source/pull/1#discussion_r62979358>:
> 
>>  	PACKAGES:=\
>> -		kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
>> -		block-mount kmod-usb-core kmod-usb-dwc2 kmod-usb2 kmod-usb-ohci \
>> -		kmod-mt76
>> +		kmod-mt76 kmod-scsi-core kmod-scsi-generic kmod-mmc kmod-sdhci \
>> +		block-mount kmod-usb-core kmod-usb2 kmod-usb-ohci kmod-usb-serial \
>> +		kmod-usb-serial-option kmod-usb-serial-wwan comgt wwan
> 
> Understood. The SDHC modules should remain though, as the connector is
> soldered on.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/lede-project/source/pull/1/files/aa6b387161fe65b44216824a4c2c0f9f71a86ffd#r62979358>
> 

but its the wrong module. did you test the connector ?